aiida/manage/configuration/profile.py

```diff
@@ -17,23 +17,12 @@
 from aiida.common import exceptions
 
 from .options import parse_option
-from .settings import DAEMON_DIR, DAEMON_LOG_DIR
 
 if TYPE_CHECKING:
     from aiida.orm.implementation import StorageBackend
 
 __all__ = ('Profile',)
 
-CIRCUS_PID_FILE_TEMPLATE = os.path.join(DAEMON_DIR, 'circus-{}.pid')
```

Comment:
I guess it's unlikely that someone (plugin) used these constants...
ok for me to remove them, but perhaps run a quick search on github just to make sure

Reply:
Did that indeed, and nothing came up.